00-programming-fundamentals/exercises/merge-conflict-remote.md

```diff
@@ -0,0 +1,102 @@
+   7. Breakout Room 2:  Rhino
+   8. Breakout Room 2:  Curassow
+2. **Fork the activity repo**: One person in your group should fork the carnival-recipes repo and then add everyone else as a collaborator.
+3. **Clone the forked repo**: Everyone else in your group should clone the forked repository.
+4. **Find your recipe scrap**: Each group is assigned six recipe scraps that should all be part of the same recipe. You can split these scraps up amongst your group however you wish. The scraps assigned to each group are listed in the carnival-recipes repo. If your group has fewer than six people someone may need to take multiple scraps.
+5. **Add your recipe scrap**: Copy the contents of your recipe scrap into the recipe file and then add and commit your changes to the recipe file. After completing this step each person in your group should have **different** content within the recipe file in their local repository.
+
+**Do not** push your changes to GitHub yet! If you're finished early with this step help your groupmates.
+
+### Building the recipe
+
+Once everyone in your group has completed the baseline setup instructions above you're ready to start building the recipe together.
+
+#### General advice
+
+There are a few things to look out for when working through this activity:
+
+- If you're still unsure about what a merge conflict is, or how to resolve one please review the [Merge Conflicts curriculum](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/git-merge-conflicts.md).
+- You can review the [slides](https://docs.google.com/presentation/d/15OFyyoNJevzk7IpdZCP6VwMLju0MVS2pv0qeETLOa6Q/edit?usp=sharing) and [example walkthrough video](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=ec7dbf60-e59b-4317-a4a5-abbe002eb7ea) if you are unsure about how to complete the activity.
+- The end result should be a real, usable recipe. The original recipe files were all complete works from various recipe websites online.
+- If you find that the result of one of your merges is missing critical ingredients or any numbered steps, you should go back and double check that you didn't lose any lines while resolving a merge conflict.
+- Your recipe scraps may have duplicate lines. We've added these in to more accurately portray the full range of potential merge conflicts. **Important**: you should get rid of any duplicate lines while merging.
+- There's no "right" way to do this activity, all of the strategies listed below are valid ways to build the recipe file.
+- Have fun!
+
+#### Recipe building process
+
+Because each member of your group now has a different version of the recipe file in their local repository, building the complete recipe by merging those versions together will result in merge conflicts. To handle those merge conflicts sensibly your group should use the following process to construct the final recipe file:
+
+1. Split your group into two teams of two or three.
```

Comment:
Just thought the original wording made it sound like the students needed to make a choice about group size, which felt a little confusing to me.
```suggestion
1. Split your group into two teams of three. (There may be a group of two if your group only has 5 people.)
```

Reply:
This does make me realize we may need a specific strategy for what to do about the one group of 5 that will exist.
Also makes me realize it will be especially helpful with this activity if we know ahead of time whether anyone will be absent.